src/shared/components/InputBase/index.ts

```diff
@@ -1,5 +1,3 @@
-import InputBase, { InputBaseProps } from './InputBase'
-import { LabelText } from './InputBase.styles'
+export * from './InputBase'
 
-export { LabelText, InputBase as default }
-export type { InputBaseProps }
+export * from './InputBase.styles'
```

Comment:
In that case maybe let's pick what we export because `styles` file will have a lot of things not needed in global scope